database/sqlite/sqlite_functions.c

```diff
@@ -2208,3 +2212,263 @@ void sql_load_node_id(RRDHOST *host)
 
     return;
 };
+
+/*
+ * Store a gap in the database
+ */
+int sql_store_gap(
+    uuid_t *gap_uuid,
+    char *host_mguid,
+    int t_delta_start,
+    int t_delta_first,
+    int t_delta_end,
+    char *status)
+{
+    sqlite3_stmt *res = NULL;
+    int rc, param = 0;
+
+    if (unlikely(!db_meta)) {
+        if (default_rrd_memory_mode != RRD_MEMORY_MODE_DBENGINE)
+            return 0;
+        error_report("Database has not been initialized");
+        return SQLITE_ERROR;
+    }
+
+    if (unlikely(!res)) {
+        rc = prepare_statement(db_meta, SQL_STORE_GAP, &res);
```

Comment:
In the previous PR @stelfrag commented [this](https://github.com/netdata/netdata/pull/12556/files#r879024601) and I reinforce the requirement, please, change the function.